src/components/faq/calendar/Calendar.js

```diff
@@ -1,18 +1,78 @@
 import React from 'react';
-
 import styled from "styled-components";
+import { CalendarActiveCSS, CalendarInactiveCSS, CalendarTitleCSS, CalendarDesCSS, CalendarTagCSS} from "./CalendarCSS"
 
-import { colors } from "../../colors/index";
-
-// import css later
+import CalendarTagImage from '../../../assets/tag_filled.svg';
 
 // Calendar
-function Calendar() {
+const CalendarComponent = (props) => {
```

Comment:
Deconstruct props here like this 
`const CalendarComponent = ({description, active, date, title}) => {`